### DISCUSSION TITLE
vertical bar?

### DISCUSSION THREAD

**Discussion by paulokanda** [geovens/gInk#125] [Q&A]
Hi,
first of all, great app, congrats!! Very usefull in pandemic times when everyone is screen teaching.

Can I change config to present bar in vertical mode and is it possible to narrow bar a little bit?

best
Paulo Kanda
Brazil

**Comment by pubpub-zz** [comment]
Hi, I've developped a fork where those two options have been developped
https://github.com/pubpub-zz/ppInk

**Reply by pubpub-zz** [reply]
@paulokanda , is it what you are looking for?
